_data/other.yml

```diff
@@ -72,6 +72,14 @@ websites:
       tfa: No
       twitter: Azendoo
 
+    - name: BasisCode Compliance
+      url: http://basiscode.com
+      img: http://basiscode.com/wp-content/uploads/2016/10/BasisCode-Compliance.png
```

Comment:
See [FAQ_New_TFA_SITE](https://github.com/2factorauth/twofactorauth/wiki/FAQ_New_TFA_SITE) for how to correctly add an image.